index.js

```diff
@@ -1,42 +1,13 @@
 require("dotenv").config();
 checkConfig();
-const express = require("express");
-const app = express();
 const mongoose = require("mongoose");
-const cors = require("cors");
```

Comment:
I believe that we cannot remove basic libraries, like express or cors

Reply:
Cors is removed here because it is in server.js. We don't need it twice.

Reply:
ok, got it